docs/releases/1.9.txt

```diff
@@ -124,6 +124,12 @@ levels of graceful degradation.
 
 .. _YUI's A-grade: http://yuilibrary.com/yui/docs/tutorials/gbs/
 
+PostgreSQL backend renamed
```

Comment:
Please add a "Database backends" section to minor features and include this there.